buildstockbatch/workflow_generator/residential/latest/__init__.py

```diff
@@ -0,0 +1,10 @@
+__version__ = "2024.07.19"
+version_info = {
+    "version": __version__,
+    "version_description": """
+Version information for the residential workflow generator.
+This version is compatible with ResStock v3.2.0 and later.
```

Comment:
Not true, right? Compatible with ResStock > v3.2.0.

Reply:
Correct. Thanks for catching this.